6.1.flatlandia.py

```diff
@@ -0,0 +1,21 @@
+
+
+def find_special_state(states):
+    # define the sets of letters in the same line in keyboard (qwerty) in a list
+    list_of_sets = [set("qwertyuiop"), set("asdfghjkl"), set("zxcvbnm")]
+    # find the special state
+    for state in states:
+        state_set = set(state.lower().rstrip('\n'))
+        for set_of_letters in list_of_sets:
```

Comment:
You should aim to write everything in a single line, even if you can. 
This link https://towardsdatascience.com/you-should-stop-craving-for-python-one-liners-5dbf8e0de23d explains it well, even though I disagree with how important memory and runtime are in most python programs.